### PR TITLE
Dynamic data tour trip

### PR DESCRIPTION
**Pull request by Joanartdacosta**
props from different components
tour Id page - load info from mongodb

### CODE REVIEW

File: app/pages/trips/[tripId].js [hunk 1]
```diff
@@ -18,7 +12,21 @@ export default function TripDetailsPage(props) {
         description={NAVBAR_LINKS["trips"]["description"]}
       />
       <div className="roboto max-w-5xl m-auto pt-20 pb-20">
-        <MenuDetailsId />
+        <MenuDetailsId
+          // access={props.selectedTrip.access}
```
Comment (doublethink13) [inline app/pages/trips/[tripId].js:16]: delete comment

File: app/pages/tours/[tourId].js [hunk 2]
```diff
@@ -18,7 +12,26 @@ export default function TourDetailPage(props) {
         description={NAVBAR_LINKS["tours"]["description"]}
       />
       <div className="roboto max-w-5xl m-auto pt-20 pb-20">
-        <MenuDetailsId />
+        <MenuDetailsId
+          duration={props.selectedTour.duration}
```
Comment (doublethink13) [inline app/pages/tours/[tourId].js:16]: alphabetical order

File: app/components/common/menu_id/path/PathOptions.js [hunk 3]
```diff
@@ -16,11 +15,30 @@ export default function PathOptions() {
...
+        <div class="py-5 border-b border-gray-200 dark:border-gray-700">
+          <p class="mb-2 text-gray-500 dark:text-gray-400">
+            {props.description1}
+          </p>
+          <p class="mb-2 text-gray-500 dark:text-gray-400">
+            {props.waydescription}
+          </p>
+        </div>
+        <Path
+          label={PATH_OPTIONS_LABELS.pathOption1}
```
Comment (doublethink13) [inline app/components/common/menu_id/path/PathOptions.js:27]: alphabetical order

File: app/components/common/menu_id/path/Path.js [hunk 4]
```diff
@@ -24,7 +24,15 @@ export default function Path(props) {
           <Arrow />
         </button>
 
-        {show && <Info onClick={close} />}
+        {show && (
+          <Info
+            onClick={close}
```
Comment (doublethink13) [inline app/components/common/menu_id/path/Path.js:29]: alphabetical order

File: app/components/common/menu_id/MenuDetailsId.js [hunk 5]
```diff
@@ -6,31 +6,80 @@ import PathOptions from "./path/PathOptions";
...
 
-export default function MenuDetailsId() {
+export default function MenuDetailsId(props) {
   const [isOpen, setIsOpen] = useState(false);
-  const [info, setInfo] = useState(<GeneralSection />);
+  const [info, setInfo] = useState(
+    <div>
+      {" "}
+      <GeneralSection
+        duration={props.duration}
```
Comment (doublethink13) [inline app/components/common/menu_id/MenuDetailsId.js:17]: please check all props alphabetical order

**Comment by doublethink13** [comment]
deployment is failing. please check that as well

File: app/pages/tours/[tourId].js (hunk 2 above)
Comment (Joanartdacosta) [reply app/pages/tours/[tourId].js]: done!

File: app/components/common/menu_id/path/Path.js (hunk 4 above)
Comment (Joanartdacosta) [reply app/components/common/menu_id/path/Path.js]: done!

File: app/components/common/menu_id/MenuDetailsId.js (hunk 5 above)
Comment (Joanartdacosta) [reply app/components/common/menu_id/MenuDetailsId.js]: well noted, thanks!
done

File: app/pages/trips/[tripId].js (hunk 1 above)
Comment (Joanartdacosta) [reply app/pages/trips/[tripId].js]: done!

**Comment by Joanartdacosta** [comment]
> deployment is failing. please check that as well

File: app/components/common/menu_id/classification/Classification.js [hunk 6]
```diff
@@ -1,23 +1,128 @@
...
+  let percentageSupport = props.percentageSupport;
 
-export default function Classification() {
   return (
     <div className="p-8 m-auto">
-      {CLASSIFICATIONS.map((item) => {
-        return (
-          <div key={item.id}>
+      <div>
+        {props.percentageAcessibility ? (
```
Comment (doublethink13) [inline app/components/common/menu_id/classification/Classification.js:15]: you should either not create a var called percentageAcessibility or use it in this check

Comment (doublethink13) [reply app/components/common/menu_id/classification/Classification.js:15]: what i mean here is

on line 4, you create a var called `acessibility` from `props.percentageAcessibility`

then, on line 15, you use `props.percentageAcessibility`

but then, on line 23, you use `acessibility`

i dont see the need of creating `acessibility`

you should just use always `props.percentageAcessibility`

Comment (Joanartdacosta) [reply app/components/common/menu_id/classification/Classification.js:7]: done!

**Pull request merged**